Split/Common/Utils/CastUtils.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Any+Extensions.swift
+//  Split
+//
+//  Created by Javier L. Avrudsky on 16/11/2018.
+//  Copyright © 2018 Split. All rights reserved.
+//
+// This class was created as a workaround to fix some convertion problems
+// related to Int datatypes until doing a full refactor.
+// ToDo: Refactor Int usages
+
+import Foundation
+
+
+class CastUtils {
+    static func anyToInt64(value: Any?) -> Int64? {
+        if let val = value as? NSNumber {
+            if val.intValue <= Int64.max {
+                return Int64(truncating: val)
+            }
+        }
+        return nil
```

Comment:
are we ok returnin nill?

Reply:
I think is the best option for the case the cast fails. What do you think about that? Do you have any suggestion?